### ISSUE TITLE
Alex flags "slave" but not "master"

### ISSUE THREAD

**Issue by robdodson** [get-alex/alex#298]
### Subject of the issue

Alex will flag the word "slave" but not the word "master"

### Your environment

*   **OS**: macOS Catalina
*   **Packages**: alex (current version)
*   **Env**: node v12.12.0, npm 6.11.3

### Steps to reproduce

Create an `index.md` file with:

```
hello world

master

slave

goodbye
```

Run `npx alex index.md`

### Expected behaviour

Alex warns about the use of master and slave

### Actual behaviour

Alex warns about the use of slave

**Comment by wooorm** [comment]
Hi Rob! Thanks for the suggestion, and welcome! I think warning for master on its own is a good idea.

If you/someone is interested in working on it, solving it would mean a PR to [retext-equality](https://github.com/retextjs/retext-equality)!

**Comment by robdodson** [comment]
Hey @wooorm 

I tried to add master to retext-equality but it throws because it's a duplicate. I believe it's because `master and slave` is already defined as a `-type: and` pattern maybe?

Here's what I added to the top of [race.yml](https://github.com/retextjs/retext-equality/blob/5ebc3d8565b50fda9a291986f3eba5f06e7dcb41/data/en/race.yml):

```yml
- type: basic
  considerate:
    - primary
  inconsiderate:
    - master
```

Here's the error:

```bash
Error: Refrain from multiple entries:
  master
    at generateLanguage (/Users/robdodson/Developer/retext-equality/script/generate.js:120:11)
```

Also it looks like "slave" is already flagged by [retext-profanities](https://github.com/retextjs/retext-profanities). Would it be easier to add "master" to retext-profanities as well?

**Comment by wooorm** [comment]
Thanks for working on this! Yes, indeed, the other pattern probably interferes. Maybe time to remove it? And replace it with patterns to warn about master and slave, separately? I think it’s the only *and* pattern though, so might cause problems with test coverage, if so, I can handle removing dead code.

My personal opinion is that I don’t think master really makes sense in retext-profanities: that’s more for things that are (sometimes) slurs. Master isn’t really a slur. I think it’s okay for `slave` to be matched in both projects though. What are your thoughts?

**Comment by robdodson** [comment]
> Thanks for working on this! Yes, indeed, the other pattern probably interferes. Maybe time to remove it? And replace it with patterns to warn about master and slave, separately?

Yeah I think warning about 'master' and 'slave' using the `basic` type would work. It should cover all scenarios.

> I think it’s the only and pattern though, so might cause problems with test coverage, if so, I can handle removing dead code.

I would appreciate that :)
